### DISCUSSION TITLE
Using proper Validator Facade in Docs

### DISCUSSION THREAD

**Discussion by Zakarialabib** [livewire/livewire#6216] [7. Bugs (archived)]
Hello there is a mistake in [validation.md in docs reference ](https://github.com/livewire/livewire/blob/ad33d14f9ca37a74f270b2495ec8f99aa3e29ba0/docs/validation.md?plain=1#L394)

in the perfect scenario it should be use Illuminate\Support\Facades\Validator;

**Comment by ousid** [comment]
You could open a PR suggesting your modification.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147
